.github/workflows/wheel-builder.yml

```diff
@@ -24,7 +25,11 @@ jobs:
     steps:
       - run: ${{ matrix.PYTHON.PATH }} -m venv .venv
       - name: Install Python dependencies
-        run: .venv/bin/pip install -U pip wheel cffi setuptools-rust
+        run: .venv/bin/pip install -U pip wheel setuptools-rust
+      - name: Install cffi
+        run: .venv/bin/pip install -U cffi
+        if: contains(matrix.PYTHON.VERSION, 'pypy') == false
```

Comment:
Why is this required?

Reply:
It may not be but cffi is still built into pypy so installing another version is, at best, a no op right?

Reply:
Yes, but we were producing pypy wheels already here :-)

Reply:
Just because it was working doesn't make it correct 😄. I can revert this if you prefer, but unless things have changed with cffi (which maybe it has!) installing cffi on pypy historically could potentially break the built-in cffi.

Reply:
I have a soft preference for reverting, on readability grounds.